src/LessQL/Database.php

```diff
@@ -17,7 +17,9 @@ function __construct( $pdo ) {
 		// required for safety
 		$pdo->setAttribute( \PDO::ATTR_ERRMODE, \PDO::ERRMODE_EXCEPTION );
 		$this->pdo = $pdo;
-
+        if ($this->pdo->getAttribute(\PDO::ATTR_DRIVER_NAME) === 'pgsql') {
+            $this->setIdentifierDelimiter('');
```

Comment:
or none, but still better than the not working ```

Reply:
anyhow, I tested it and used it and like this it is working, using the backticks it is not, but seems like this repo is abandoned

Reply:
blank is better than a backtick yes. a double-quote would work for others that don't use lower_case table names since pg will fold UpperCase tables into lowercase.

and yes, unfortunately, this repo seems to be abandoned.

Reply:
done